--- before_install.sh	2017-10-19 20:46:45.934796715 +0000
+++ before_install.sh	2017-10-19 16:12:20.192772175 +0000
@@ -2,10 +2,10 @@
 
 set -e
 
-sudo dpkg -l | grep geos
+#sudo dpkg -l | grep geos
 sudo apt-get purge -y libgeos* libspatialite*
-find  /etc/apt/sources.list.d
-sudo mv /etc/apt/sources.list.d/pgdg* /tmp
+#find  /etc/apt/sources.list.d
+#sudo mv /etc/apt/sources.list.d/pgdg* /tmp
 #sudo apt-get remove postgis libpq5 libpq-dev postgresql-9.1-postgis postgresql-9.2-postgis postgresql-9.3-postgis postgresql-9.1 postgresql-9.2 postgresql-9.3 libgdal1
 #sudo apt-get remove -y postgresql-9.1
 sudo add-apt-repository -y ppa:ubuntugis/ubuntugis-unstable
@@ -60,7 +60,7 @@
 unzip c6d36f3db5e4.zip
 cd chchrsc-kealib-c6d36f3db5e4/trunk
 cmake . -DBUILD_SHARED_LIBS=ON -DCMAKE_BUILD_TYPE=Release -DCMAKE_INSTALL_PREFIX=/usr -DHDF5_INCLUDE_DIR=/usr/include -DHDF5_LIB_PATH=/usr/lib -DLIBKEA_WITH_GDAL=OFF
-make -j4
+make -j$(nproc)
 sudo make install
 cd ../..
 sudo ldconfig
